opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -296,8 +311,18 @@ impl<R: RuntimeChannel> BatchLogProcessor<R> {
                 }
             }
         }));
+
+        let dropped_logs_metric = global::meter("opentelemetry")
```

Comment:
I'd suggest holding off from exposing these via OTel metrics.
1. Fetching meter from global only works if user has already set a global meter provider. There is no document guiding them to setup Metrics signal before logs/traces, so by default, this is not guaranteed to work.
2. We cannot guarantee the name of the metric nor the name of the attributes. OTel spec/convention is planning to add one for the same, and we need to wait until it lands (We can offer OTel Rust as one of the language to prototype this to support spec/semantic convention)